### DISCUSSION TITLE
Small fix to keyboard operation

### DISCUSSION THREAD

**Discussion by Jay1** [spacedriveapp/spacedrive#1898] [Feature Request]
Good evening, 

When you press Delete on your keyboard with files selected, the confirmation prompt does not focus properly. It's impossible (as far as I can tell) to press enter and just delete the files. Pressing tab multiple times or using arrows still manipulate the files behind. 

Thanks, 
Jay

**Comment by utkubakir** [comment]
Created an issue for this, thanks Jay!

**Reply by utkubakir** [reply]
Fixed
